src/pages/docs/platform/architecture/access-network.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Access network
+meta_description: "Understand how Ably's access network provides reliable, low-latency connectivity to its platform from anywhere in the world."
+---
+
+The access network serves as the entry point to Ably's realtime infrastructure and is core to the platform's ability to deliver reliable, low-latency messaging services globally.
+
+## Why the access network matters
+
+The access network must ensure that clients can reliably connect to the closest available datacenter, providing optimal performance whilst maintaining resilience against various types of failures. Given that Ably is mission-critical for many applications, the reliability of the access network is essential; any connectivity issues can potentially impact thousands or millions of end users, making the design and operation of the access network a key element of Ably's infrastructure.
+
+## Access network architecture
+
+Ably's access network is designed to provide reliable, low-latency connectivity to the Ably platform from anywhere in the world. The architecture leverages multiple AWS services and custom infrastructure components to ensure resilience, performance, and security.
+
+### Global deployment across multiple regions
+
+Ably operates in multiple regions around the world, with servers distributed across more than 15 physical datacenters within Amazon's cloud infrastructure. This global distribution ensures that clients can connect to a nearby datacenter, minimizing the latency between the client and the entry point to Ably's network.
```

Comment:
```suggestion
Ably operates in multiple regions around the world, with servers distributed across more than 15 physical datacenters within the AWS network. This global distribution ensures that clients can connect to a nearby datacenter, minimizing the latency between the client and the entry point to Ably's network.
```

Reply:
(commit references in Matt's comments above - thanks for providing the info)